web/app/view/AttributesAliasesController.js

```diff
@@ -0,0 +1,94 @@
+ */
+
+Ext.define('Traccar.view.AttributesAliasesController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.attributesAliases',
+
+    requires: [
+        'Traccar.view.AttributeAliasDialog',
+        'Traccar.model.AttributeAlias'
+    ],
+
+    init: function () {
+        var admin = Traccar.app.getUser().get('admin');
+        this.lookupReference('deviceField').setStore(admin ? 'AllDevices' : 'Devices');
+        this.lookupReference('toolbarAddButton').setDisabled(true);
+        this.lookupReference('toolbarEditButton').setDisabled(true);
+        this.lookupReference('toolbarRemoveButton').setDisabled(true);
+        this.getView().getStore().loadData([], false);
+    },
+
+    onAddClick: function () {
+        var attributeAlias, dialog, deviceId;
+        attributeAlias = Ext.create('Traccar.model.AttributeAlias');
+        attributeAlias.store = this.getView().getStore();
+        deviceId = this.lookupReference('deviceField').getValue();
+        attributeAlias.set('deviceId', deviceId);
+        dialog = Ext.create('Traccar.view.AttributeAliasDialog');
+        dialog.down('form').loadRecord(attributeAlias);
+        dialog.show();
+    },
+
+    onEditClick: function () {
+        var attributeAlias, dialog;
+        attributeAlias = this.getView().getSelectionModel().getSelection()[0];
+        dialog = Ext.create('Traccar.view.AttributeAliasDialog');
+        dialog.down('form').loadRecord(attributeAlias);
+        dialog.show();
+    },
+
+    onRemoveClick: function () {
+        var attributeAlias = this.getView().getSelectionModel().getSelection()[0];
+        Ext.Msg.show({
+            title: Strings.sharedAttributeAlias,
+            message: Strings.sharedRemoveConfirm,
+            buttons: Ext.Msg.YESNO,
+            buttonText: {
+                yes: Strings.sharedRemove,
+                no: Strings.sharedCancel
+            },
+            scope: this,
+            fn: function (btn) {
+                var store = this.getView().getStore();
+                if (btn === 'yes') {
+                    store.remove(attributeAlias);
+                    store.sync();
+                }
+            }
+        });
+    },
+
+    onSelectionChange: function (selected) {
+        var disabled = !this.lookupReference('deviceField').getValue();
+        this.lookupReference('toolbarAddButton').setDisabled(disabled);
+        disabled = selected.length === 0 || !this.lookupReference('deviceField').getValue();
+        this.lookupReference('toolbarEditButton').setDisabled(disabled);
+        this.lookupReference('toolbarRemoveButton').setDisabled(disabled);
+    },
+
+    onDeviceChange: function (combobox, newValue, oldValue) {
+        this.onSelectionChange('');
+        if (newValue !== null) {
+            this.getView().getStore().getProxy().setExtraParam('deviceId', newValue);
+            this.getView().getStore().load();
+        } else {
+            this.getView().getStore().loadData([], false);
```

Comment:
Same as above. Also, even if you use this method, you don't have to provide second parameter. It defaults to "false".